@fregante Thank you!

I got such error when trying to build:

```
ERROR in ./node_modules/webext-permission-toggle/index.js
Module not found: Error: Can't resolve 'webext-permissions' in '/Users/enix/workspace/Gitako/node_modules/webext-permission-toggle'
 @ ./node_modules/webext-permission-toggle/index.js 5:0-62 66:20-44
 @ ./src/background.ts
```

Then I noticed the reason is that `webext-permissions`'s package.json file has no `main` field. If I manually add the field as `distribution/index.js`, the build succeeded.

Is it expected to omit the `main` field?

ref: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -99,6 +99,7 @@ module.exports = {
   resolve: {
     extensions: ['.ts', '.tsx', '.js', '.jsx', '.json'],
     modules: [srcPath, packagesPath, 'node_modules'],
+    mainFields: ['main', 'exports'],
   },
   module: {
     rules: [
@@ -113,10 +114,28 @@ module.exports = {
         sideEffects: false,
       },
       {
-        test: /\.m?js$/,
+        test: /\.[cm]?js$/,
         loader: 'babel-loader',
         // Transpile as least files under node_modules
-        include: /node_modules\/(webext-content-scripts|webext-detect-page|superstruct)\/.*\.m?js$/,
+        include: new RegExp(
+          [
+            ``,
+            `node_modules`,
+            `(${[
+              `superstruct`,
+              `webext-alert`,
+              `webext-content-scripts`,
+              `webext-detect-page`,
+              `webext-detect`,
+              `webext-dynamic-content-scripts`,
+              `webext-events`,
+              `webext-permission-toggle`,
+              `webext-permissions`,
+              `webext-tools`,
+            ].join('|')})`,
+            ``,
+          ].join('/'),
+        ),
         options: {
           cacheDirectory: true,
         },

